src/services/Initialization.cpp

```diff
@@ -0,0 +1,22 @@
+#include "Initialization.hh"
+
+#include "services\ServiceLocator.hh"
+#include "services\impl\JsonFileConfiguration.hh"
+#include "services\impl\LeaderboardManager.hh"
+
+namespace ra {
+namespace services {
+
+void Initialization::RegisterServices(const std::string& sHomeDir, const std::string& sClientName)
+{
+    std::string sFilename = sHomeDir + "RAPrefs_" + sClientName + ".cfg";
+    auto* pConfiguration = new ra::services::impl::JsonFileConfiguration();
+    pConfiguration->Load(sFilename);
+    ra::services::ServiceLocator::Provide<ra::services::IConfiguration>(pConfiguration);
```

Comment:
Here's where we register an implementation for an interface.

In the future, if we wanted to store preferences on the server, we'd just change the implementation from `JsonFileConfiguration` to `ServerJsonConfiguration` or something similar.